### DISCUSSION TITLE
3.1.0

### DISCUSSION THREAD

**Discussion by sfmskywalker** [elsa-workflows/elsa-studio#180] [Announcements]
## What's Changed
* Use Central Package Management by @sfmskywalker in https://github.com/elsa-workflows/elsa-studio/pull/144
* Using same label and icon for the same import/export actions by @dix in https://github.com/elsa-workflows/elsa-studio/pull/132
* Fix typo for GetUIHintsHandler by @Huzarensalade in https://github.com/elsa-workflows/elsa-studio/pull/135
* Add Workflow Instance List Filter by @sfmskywalker in https://github.com/elsa-workflows/elsa-studio/pull/146
* Dispaly input and output in the workflow instance by @MariusVuscanNx in https://github.com/elsa-workflows/elsa-studio/pull/143
* Fixed null reference bug and improved input output display by @MariusVuscanNx in https://github.com/elsa-workflows/elsa-studio/pull/150
* Workflow Instance Filter Improvements by @sfmskywalker in https://github.com/elsa-workflows/elsa-studio/pull/151
* Added workflow instance and bulk cancellation to the UI by @raymonddenhaan in https://github.com/elsa-workflows/elsa-studio/pull/148
* Prevent duplicate 'Default' switch cases by @sfmskywalker in https://github.com/elsa-workflows/elsa-studio/pull/155
* Added Json specific code editor component by @raymonddenhaan in https://github.com/elsa-workflows/elsa-studio/pull/158
* Features/input descriptor readonly by @jdevillard in https://github.com/elsa-workflows/elsa-studio/pull/139
* Add search functionality to WorkflowDefinitionList by @sfmskywalker in https://github.com/elsa-workflows/elsa-studio/pull/166
* Update MudStack row property in ActivityExecutionsTab by @sfmskywalker in https://github.com/elsa-workflows/elsa-studio/pull/165
* Update WorkflowInstanceDetails with ActivityExecutionRecord by @sfmskywalker in https://github.com/elsa-workflows/elsa-studio/pull/168
* UI fixes for filters by @MariusVuscanNx in https://github.com/elsa-workflows/elsa-studio/pull/170
* fix: remove useless input port in start node. by @zergmk2 in https://github.com/elsa-workflows/elsa-studio/pull/169
* Reverted tooltip on search inputs, increased the debounce interval, temporary disabled export UI functionality by @MariusVuscanNx in https://github.com/elsa-workflows/elsa-studio/pull/171
* Temporary disabled edit button, increased debounce interval by @MariusVuscanNx in https://github.com/elsa-workflows/elsa-studio/pull/172
* Added back edit button by @MariusVuscanNx in https://github.com/elsa-workflows/elsa-studio/pull/173
* Update Elsa API Client package reference by @sfmskywalker in https://github.com/elsa-workflows/elsa-studio/pull/163
* Improve data display in workflow viewer by @sfmskywalker in https://github.com/elsa-workflows/elsa-studio/pull/176
* Add string trimming extensions and usage by @sfmskywalker in https://github.com/elsa-workflows/elsa-studio/pull/179

## New Contributors
* @dix made their first contribution in https://github.com/elsa-workflows/elsa-studio/pull/132
* @Huzarensalade made their first contribution in https://github.com/elsa-workflows/elsa-studio/pull/135
* @raymonddenhaan made their first contribution in https://github.com/elsa-workflows/elsa-studio/pull/148
* @zergmk2 made their first contribution in https://github.com/elsa-workflows/elsa-studio/pull/169

**Full Changelog**: https://github.com/elsa-workflows/elsa-studio/compare/3.0.5...3.1.0

<hr /><em>This discussion was created from the release <a href='https://github.com/elsa-workflows/elsa-studio/releases/tag/3.1.0'>3.1.0</a>.</em>

**Comment by glime-ai** [comment]
The provided information appears to be a changelog for the Elsa Studio project, detailing changes made between versions 3.0.5 and 3.1.0. It lists various pull requests that have been merged, improvements made, bugs fixed, and new contributors to the project. However, there is no specific question or request for action in your message.

If you need information on a specific change or if you have any other questions related to the Elsa Studio release 3.1.0, please provide more details so I can assist you accordingly.
